projects/demo/src/modules/components/icon/index.html

```diff
@@ -34,6 +34,13 @@
             [component]="9 | tuiComponent"
             [content]="9 | tuiExample: 'html,ts'"
         />
+
+        <tui-doc-example
+            id="dynamic-construct-svg"
+            heading="Dynamic construct svg"
```

Comment:
Let's call it `Inline Svg`

The such naming was used for the previous generation of `Icon`-component:
https://taiga-ui.dev/v3/icons/bundled#all
I think it was rather self-descriptive.